<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QWidget" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>114</red>
        <green>137</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>114</red>
        <green>137</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>114</red>
        <green>137</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>114</red>
        <green>137</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="Jur_And_User">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>7</y>
     <width>130</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>权限/用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateTime">
   <property name="geometry">
    <rect>
     <x>617</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>2017/02/01 13:28</string>
   </property>
  </widget>
  <widget class="QLabel" name="DisLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>登陆日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadLog_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>登陆日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NGButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>NG日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="USBOuput">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>USB输出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>360</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="Stack">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>40</y>
     <width>791</width>
     <height>311</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="LoadLog">
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>-53</x>
       <y>1</y>
       <width>817</width>
       <height>321</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>年/月/日/时/分</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作人</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="NGLog">
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>438</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AR PL UKai CN</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>321</width>
       <height>301</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom:1px solid #000;
background-color: rgb(212, 230, 202);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Proportion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>178</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NG数占比检测数比例:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom:1px solid #000;
background-color: rgb(212, 230, 202);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom:1px solid #000;
background-color: rgb(212, 230, 202);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="EndTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>178</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>日志结束时间:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>225</y>
        <width>178</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>NG数:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>178</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>产品名称:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Number">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>178</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>产品编号:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>225</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom:1px solid #000;
background-color: rgb(212, 230, 202);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>130</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2017.1.19</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="StartTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>178</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>日志开始时间:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="TestNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>178</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>检测数:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom:1px solid #000;
background-color: rgb(212, 230, 202);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>130</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2017.1.19</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
